integrations/mistral/src/haystack_integrations/components/generators/mistral/chat/chat_generator.py

```diff
@@ -20,9 +20,21 @@ class MistralChatGenerator(OpenAIChatGenerator):
     directly to this component via the `**generation_kwargs` parameter in __init__ or the `**generation_kwargs`
     parameter in `run` method.
```

Comment:
```suggestion
    directly to this component via the `generation_kwargs` parameter in `__init__` or the `generation_kwargs`
    parameter in `run` method.
```
These are not actual kwargs, but dicts.